### ISSUE TITLE
x/build/cmd/gopherbot: changes are not seen for refs/meta/config branch

### ISSUE THREAD

**Issue by andybons** [golang/go#23865]
This is due to a Gerrit config problem. gobot can’t see a change (https://go-review.googlesource.com/c/go/+/94016) and so it keeps erroring out and causing a 30s backoff.

Is there a reason why we hide our refs/meta/config from the public?

/cc @bradfitz

**Comment by bradfitz** [comment]
> Is there a reason why we hide our refs/meta/config from the public?

I don't think we ever made an explicit decision to do that. Sure that wasn't the default as it was given to us?

**Comment by andybons** [comment]
It’s definitely possible. But the person was able to create a change from it so is it visible via git? That would be weird.

**Comment by bradfitz** [comment]
> But the person

What person?

**Comment by andybons** [comment]
The author of the change linked above.

**Comment by bradfitz** [comment]
Well that's bizarre. No clue.
